.gitignore

```diff
@@ -158,3 +158,4 @@ schema/sssom_datamodel.py
 sssom/internal_context.py
 sssom/sssom_datamodel.py
 *.rej
+poetry.lock
```

Comment:
@hrshdhgd if we're giving up on poetry locks, can we put that into a different PR? I don't want people saying later that I snuck this in :)

Reply:
Fair enough. The tests will fail for now but I think I have a solution.

Reply:
Yes, I mean we can solve the problem in a separate PR, merge it into master, then just update this PR on top of it. Thanks!

Reply:
Done: https://github.com/mapping-commons/sssom-py/pull/399